src/bundler.ts

```diff
@@ -191,14 +218,20 @@ class Bundler {
 
     const imprt = importMap.get(resolvedUrl);
     if (imprt) {
+      // If the document wasn't loaded for the import during analysis, we can't
+      // inline it.
+      if (!imprt.document) {
+        // TODO(usergenic): What should the behavior be when we don't have the
+        // document to inline available in the analyzer?
+        return;
```

Comment:
Well, I think the analyzer will need to be able to continue its work when it fails to load an import.  At least for the linter, this behavior is high value.  As for the bundler, it might be nice to be able to bundle incompletely especially in a development context.  Rather than throwing, we should consider building some kind of Errors collection we can return at the end of processing.  Of course, we can provide option to fail-fast, too.